_data/versions.yml

```diff
@@ -1,4 +1,55 @@
 v2.6:
+- title: v2.6.2
+  note: |
+    16 October 2017
+
+    Calico v2.6.2 adds fixes and enhancements for Calico/OpenStack deployments.  For Kubernetes and other integrations there is no change from v2.6.1.
+
+    - The dnsmasq packages that we provide have been upgraded so as to address various [security issues](https://github.com/projectcalico/calico/issues/1169).
+
+    - networking-calico master has been released, as 1.4.3, so as to provide support for OpenStack Ocata and later.
+
+    <div class="alert alert-danger" role="alert"><b>Important</b>: If you are using the Kubernetes datastore and upgrading from Calico v2.4.x or earlier to Calico v2.5.x or later, you must <a href="https://github.com/projectcalico/calico/blob/master/upgrade/v2.5/README.md">migrate your Calico configuration data</a> before upgrading. Otherwise, your cluster may lose connectivity after the upgrade.</div>
+
+  components:
+    felix:
+      version: 2.6.0
+      url: https://github.com/projectcalico/felix/releases/tag/2.6.0
+    typha:
+      version: v0.5.1
+      url: https://github.com/projectcalico/typha/releases/tag/v0.5.1
+    calicoctl:
+      version: v1.6.1
+      url: https://github.com/projectcalico/calicoctl/releases/tag/v1.6.1
+      download_url: https://github.com/projectcalico/calicoctl/releases/download/v1.6.1/calicoctl
+    calico/node:
+      version: v2.6.1
+      url: https://github.com/projectcalico/calico/releases/tag/v2.6.0
```

Comment:
shouldn't this be v2.6.1? Looks like it's pointing to the bad release that I cranked out a few weeks ago ... if it's incorrect, then it should probably be fixed in the v2.6.1 release below as well.